assignment3/app.js

```diff
@@ -0,0 +1,27 @@
+var http = require('http');
+var util = require('util');
+
+var secondlargest = require('./secondlargestarray');
+
+var frequency=require('./calculateFrequency');
```

Comment:
space before and after operator is expected. Eg: instead of `a=3` , write `a = 3`